starlette/websockets.py

```diff
@@ -53,6 +58,8 @@ async def receive(self) -> Message:
                 )
             if message_type == "websocket.disconnect":
                 self.client_state = WebSocketState.DISCONNECTED
+                if "code" not in message:
+                    message["code"] = 1005  # websocket spec
```

Comment:
This can also be removed from this PR, I guess?

Reply:
Yes, sure.